bazel/repositories.bzl

```diff
@@ -215,7 +215,7 @@ def _list_pl_deps(name):
                 best_url = url
         modules[best_url] = True
 
-    for repo_name, repo_config in GIT_REPOSITORY_LOCATIONS.items():
+    for _, repo_config in GIT_REPOSITORY_LOCATIONS.items():
```

Comment:
`repo_name` was causing the run-container-lint check to fail due to it being unused.